README.md

```diff
@@ -1,87 +1,161 @@
-# Builder DAO.
+# Builder DAO
 
-Builder DAO monorepo.
+Welcome to the [Builder DAO](https://builderdao.notion.site) monorepo! This [turborepo](https://turborepo.org/) uses [Yarn](https://yarnpkg.com/) as a package manager.
 
-## What's inside?
+![](https://img.shields.io/twitter/follow/TheBuilderDAO?style=social)
 
-This turborepo uses [Yarn](https://classic.yarnpkg.com/lang/en/) as a package manager. It includes the following packages/apps:
-
-### Apps, Packages, Tutorials, and Solana Programs.
-
-- apps:
-  - `web`: BuilderDAO frontend
-    - `/learn/<tutorial-name>/<md-file-name>`Renders tutorials.
-    - `/proposal/<proposal-publickey>`Renders proposals. (TODO)
-- packages:
-  - `ui`: a stub React component library shared by both `web` and `docs` applications
-  - `config`: `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
-  - `tsconfig`: `tsconfig.json`s used throughout the monorepo
-  - `md-utils`: utilities for parsing markdown files
-- tutorials:
-  - `<tutorial-name>`
-    - `/builderdao.config.js`: configuration for the tutorial
-    - `/content/<md-file>`: tutorial content
-- programs:
-  - `<solana-program-name>`: a solana program
-    - `/src`: rust source code for the program
-    - `/ts`: SDK source code
-    - `/tests/`: rust test code for the program
-    - `/__tests__`: typescript tests for the program
-
-### Build
-
-To build all apps and packages, run the following command:
+![](https://img.shields.io/github/contributors/TheBuilderDAO/kafe?style=plastic)
+![](https://img.shields.io/github/commit-activity/m/TheBuilderDAO/kafe?style=plastic)
 
-```
-yarn run build
+## What's inside?
+
+This repo contains the [Kafé learning platform](https://dev.builderdao.io) created by Builder DAO including the tutorials, web frontend, Solana programs and TypeScript SDK. 
+
+Click each heading below for details:
+
+<details>
+ <summary>/apps</summary>
+
+- `/web` - Kafé by BuilderDAO frontend
+  - `/components` - ReactJS components
+  - `/hooks` - React Hooks specific to Kafé
+  - `/pages` - Next.js API & page routes
+  - `/styles` - This project uses [TailwindCSS](https://tailwindcss.com/)
+</details>
+
+<details>
+ <summary>/dao-programs</summary>
+
+- `/migrations/deploy.js` - Deployment script
+- `/programs/<solana-program-name>` - Solana program
+  - `/src/`: Rust source code for the program
+  - `/tests/`: Rust tests for the program
+  - `/ts-sdk/`: TypeScript SDK source code
+</details>
+
+<details>
+ <summary>/packages</summary>
+
+- `/ui`- A stub React component library
+- `/config` - `eslint` configurations (includes `eslint-config-next` and `eslint-config-prettier`)
+- `/tsconfig` - TypeScript configuration used throughout the monorepo
```

Comment:
I think we should not change to `repo` there's a clear separation of the code structure between different architectures. in this monorepo things are modular and separate which gives users the to separately use our packages to build different things on top of the builderdao SDK without explicitly requiring our kafe repository.

https://monorepo.tools/

Reply:
Are you referring to the comment to change monorepo to repo in line 41, or the general comment I made above in line 3?

Reply:
**General comment** as this project is architecturally utilizing the monorepo to have modular packages which separately published and can be used as well.

Reply:
I agree with you that monorepo is technically more precise, but a developer who knows about monorepos won't be confused if the we call it a repo at the top and introduce the concept of monorepo later in the README where appropriate (maybe with a link to the monorepo.tools site). But a developer who isn't familiar with monorepos will be confused right at the top of the README (i.e. they'll pause and go, "wait what's a monorepo" and the cost of engaging with the README will become higher).

Let me know if your logaf is high on this and we can chat about it. Otherwise, let's call it a repo in the intro and introduce the concept of monorepo (with a link to monorepo.tools) in the "How to use?" section -- maybe something like, "This repo is structured as a [monorepo](https://monorepo.tools/), which keeps things modular and gives us the ability to build on top of the builderdao SDK independent of Kafé."

Reply:
Understood yeah I'm cool with it as you suggested. 🙌